postfix/ci/postfix.rake

```diff
@@ -0,0 +1,64 @@
+require 'ci/common'
+
+def postfix_version
+  ENV['FLAVOR_VERSION'] || 'latest'
+end
+
+def postfix_rootdir
```

Comment:
I'll remove it here, and I'll add that to our list of tasks to do after we do the cleanup